core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.directive.html

```diff
@@ -1,4 +1,4 @@
-<ul class="nav navbar-nav oppia-navbar-nav float-right ng-cloak"
+<ul class="navbar-desktop nav navbar-nav oppinavbar-desktopa-navbar-nav float-right ng-cloak"
```

Comment:
hm, `oppinavbar-desktopa-navbar-nav` seems incorrect. Please check.

Reply:
This is weird. Fixed!